Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -2062,8 +2065,26 @@ lemma mem_coe_supp_of_adj {v w : V} {H : Subgraph G} {c : ConnectedComponent H.c
   rw [← (mem_supp_iff _ _).mp h.1]
   exact ⟨connectedComponentMk_eq_of_adj <| Subgraph.Adj.coe <| h.2 ▸ hadj.symm, rfl⟩
 
+lemma pairwise_disjoint_supp :
+    Pairwise fun c c' : ConnectedComponent G ↦ Disjoint c.supp c'.supp := by
```

Comment:
Sorry, I should have mentioned that earlier, but can this also take `G` explicit and be put outside the `ConnectedComponent` namespace?
```suggestion
lemma pairwise_disjoint_connectedComponentSupp :
    Pairwise fun c c' : ConnectedComponent G ↦ Disjoint c.supp c'.supp := by
```

Reply:
Done:)